README.md

```diff
@@ -8,9 +8,9 @@
 
 Welcome to Braintree's iOS SDK. This library will help you accept card and alternative payments in your iOS app.
 
-A new major version of the SDK is available in beta. See the [v5 migration guide](/V5_MIGRATION.md) for details.
+A new major version of the SDK is available. See the [v5 migration guide](/V5_MIGRATION.md) for details.
```

Comment:
```suggestion
v5 is the latest major version of Braintree iOS. To update from v4, see the [v5 migration guide](/V5_MIGRATION.md).
```

Updating removing the `A new major version of the SDK is available` language will age a bit better.

Reply:
I like it!